nbgrader/coursedir.py

```diff
@@ -142,6 +142,18 @@ def _validate_notebook_id(self, proposal: Bunch) -> str:
         )
     ).tag(config=True)
 
+    instantiated_directory = Unicode(
+        'instantiated',
+        help=dedent(
+            """
+            The name of the directory that contains the version of the
```

Comment:
This help string seems wrong

Reply:
Help sting is fixed
`    source_with_tests_directory = Unicode(
        'source_with_tests',
        help=dedent(
            """
            The name of the directory that contains the version of the
            assignment that will contain both the solutions and the instantiated 
            test cases to be used by instructors. This corresponds to
            the `nbgrader_step` variable in the `directory_structure` config
            option.
            """
        )
    ).tag(config=True)`